src/app/components/experiment-data/experiment-data.component.html

```diff
@@ -1,11 +1,27 @@
 <div class="big-content">
     <div class="flex-header">
-        <h1 class="t-page-headline">Experiment Data</h1>
+        <h1 class="t-page-headline">Experiment Data
+            <button mat-icon-button class="arrow-button" (click)="onSort()"
+                aria-label="sort>=0?'arrow_upward':'arrow_downward'">
+                <mat-icon class="arrow">{{sort>=0?"arrow_upward":"arrow_downward"}}</mat-icon>
+            </button>
+        </h1>
 
-        <mat-paginator class="paginator" (page)="onPageChange($event)" [length]="collectionSize" [pageSize]="10"
-            [pageSizeOptions]="[5, 10, 25, 100]" aria-label="Select page">
+        <mat-paginator #paginatorTop class="paginator"
+            (page)="paginatorBottom.pageSize = $event.pageSize; paginatorBottom.pageIndex = $event.pageIndex; onPageChange($event)"
+            [length]="collectionSize" [pageSize]="10" [pageSizeOptions]="[5, 10, 25, 100]" aria-label="Select page">
         </mat-paginator>
     </div>
+    <div class="filter">
```

Comment:
These changes are already part of a different PR. Either clean this PR up to only contain changes relevant to the PR (as this mess of changes is hard to review) or make this dependency clear (for example by mentioning it in the PR description and marking this PR as draft until the other PR is merged).